tests/resources/example_virtualenv_project/requirements.txt

```diff
@@ -1,2 +1,2 @@
-../../..
+mlflow
```

Comment:
@harupy CI caught a change in this PR where we can no longer support requirements files that make relative path references to locations outside the directory where the requirement file is located. This seems okay to me, since relying on this kind of relative pathing is brittle / probably a bad practice. I've instead swapped in `mlflow` from PyPI here since the example project doesn't require any special MLflow code from a dev branch. Let me know what you think.

Reply:
This actually fails when the local mlflow contains a new migration script that the latest version of MLflow on PyPI doesn't.

Reply:
Got it! Added some logic to install MLflow from dev